src/main/java/org/example/reimbursement/Reimbursement.java

```diff
@@ -0,0 +1,93 @@
+package org.example.reimbursement;
+
+import java.math.BigDecimal;
+import java.util.Objects;
+
+public class Reimbursement {
+    private String Id;
+    private String ownerId;
+    private BigDecimal dailyAllowance;
+    private BigDecimal carMillage;
+
+    private ReimbursementStatus reimbursementStatus;
+
+    public String getId() {
+        return Id;
+    }
+
+    public String getOwnerId() {
+        return ownerId;
+    }
+
+    public BigDecimal getDailyAllowance() {
+        return dailyAllowance;
+    }
+
+    public BigDecimal getCarMillage() {
+        return carMillage;
+    }
+
+    public void setId(String id) {
+        Id = id;
+    }
+
+    public void setDailyAllowance(BigDecimal dailyAllowance) {
+        this.dailyAllowance = dailyAllowance;
+    }
+
+    public void setCarMillage(BigDecimal carMillage) {
+        this.carMillage = carMillage;
+    }
+
+    public ReimbursementStatus getReimbursementType() {
+        return reimbursementStatus;
+    }
+
+    public void setReimbursementType(ReimbursementStatus reimbursementStatus) {
+        this.reimbursementStatus = reimbursementStatus;
+    }
+
+    public Reimbursement(){}
```

Comment:
unused , constructor should be right after vars

Reply:
Done